### PR TITLE
Math

### PR DESCRIPTION
**Pull request by Xandr070**
I write to function for math operation with using add, dif, div and times. Also i updated readme

### CODE REVIEW

File: src/main/java/Calculator.java [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
+    /**
+     */
+    public int times(int a, int b) {
+        return a * b;
     }
-    public int solver(){
-        //TODO inser your realisation in method solver
+
+    /**
+     * Solves a mathematical expression using the provided methods.<p>
```
Comment (Andreymlc) [inline src/main/java/Calculator.java]: Delete "p" tag

File: src/main/java/Calculator.java [hunk 2]
```diff
@@ -1,17 +1,39 @@
...
+        }
     }
-    public int times(int a, int b){
-        //TODO inser your realisation in method times
+
+    /**
+     */
+    public int times(int a, int b) {
+        return a * b;
     }
```
Comment (Andreymlc) [inline src/main/java/Calculator.java]: You have mixed up the methods

File: src/test/java/CalculatorTest.java [hunk 3]
```diff
@@ -4,28 +4,42 @@
...
     }
 
     @Test
     void solver() {
-        //TODO inser your realisation in test-method solver
+        assertEquals(18, calculator.solver(4, 5, 8, 2));
+        assertEquals(11, calculator.solver(3, 4, 4, 2));
+        assertThrows(IllegalArgumentException.class,
+                () -> calculator.solver(3, 4, 4, 0), "You can't divide by 0");
     }
```
Comment (Andreymlc) [inline src/test/java/CalculatorTest.java]: These methods are not tested

File: src/main/java/Calculator.java (hunk 2 above)
Comment (Xandr070) [reply src/main/java/Calculator.java]: No, there it's alright. That error located in my tests 😁

File: src/test/java/CalculatorTest.java (hunk 3 above)
Comment (Xandr070) [reply src/test/java/CalculatorTest.java]: Oops...

Comment (Xandr070) [reply src/test/java/CalculatorTest.java]: image

**Comment by Andreymlc** [comment]
Unbelievably, bro!

**Review by Andreymlc — APPROVED** [review]
Very good

**Pull request merged**